src/components/export-gist.jsx

```diff
@@ -0,0 +1,50 @@
+/* global XMLHttpRequest:false window:false */
+/* eslint-disable no-alert */
+import React from "react";
+
+export default class ExportGist extends React.Component {
+  parseJSON(responseText) {
+    try {
+      return JSON.parse(responseText);
+    } catch (e) {
+      window.alert("Unable to create Gist");
```

Comment:
If there's an error, I would store that in `state` (since we don't use a state management library) and conditinally render a message in `render`.